nautobot/ipam/tests/test_filters.py

```diff
@@ -49,86 +50,79 @@
 )
 
 
-class VRFTestCase(FilterTestCases.FilterTestCase):
+class VRFTestCase(FilterTestCases.FilterTestCase, FilterTestCases.TenancyFilterTestCaseMixin):
     queryset = VRF.objects.all()
     filterset = VRFFilterSet
-
-    @classmethod
-    def setUpTestData(cls):
-
-        route_targets = (
-            RouteTarget.objects.create(name="65000:1001"),
-            RouteTarget.objects.create(name="65000:1002"),
-            RouteTarget.objects.create(name="65000:1003"),
-        )
-
-        tenants = Tenant.objects.filter(group__isnull=False)[:3]
-
-        vrfs = (
-            VRF.objects.create(name="VRF 1", rd="65000:100", tenant=tenants[0], enforce_unique=False),
-            VRF.objects.create(name="VRF 2", rd="65000:200", tenant=tenants[0], enforce_unique=False),
-            VRF.objects.create(name="VRF 3", rd="65000:300", tenant=tenants[1], enforce_unique=False),
-            VRF.objects.create(name="VRF 4", rd="65000:400", tenant=tenants[1], enforce_unique=True),
-            VRF.objects.create(name="VRF 5", rd="65000:500", tenant=tenants[2], enforce_unique=True),
-            VRF.objects.create(name="VRF 6", rd="65000:600", tenant=tenants[2], enforce_unique=True),
-        )
-        vrfs[0].import_targets.add(route_targets[0])
-        vrfs[0].export_targets.add(route_targets[0])
-        vrfs[1].import_targets.add(route_targets[1])
-        vrfs[1].export_targets.add(route_targets[1])
-        vrfs[2].import_targets.add(route_targets[2])
-        vrfs[2].export_targets.add(route_targets[2])
+    tenancy_related_name = "vrfs"
+
+    # Note: all assertQuerySetEqual() calls here must use ordered=False,
+    # because order_by=["name", "rd"] but name is not globally unique and rd can be null,
+    # so relative ordering of VRFs with identical name and null rd is not guaranteed.
```

Comment:
This seems like it might be better served as a part of the docstring for this test-case so that it doesn't get overlooked.